src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/model/PaymentCommand.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.model;
+
+public interface PaymentCommand {
```

Comment:
Do we need this interface?

Reply:
Will remove, just replicated the hierarchy from prod code